crosspm/helpers/package.py

```diff
@@ -95,7 +96,7 @@ def get_file(self, file_name, unpack_force=True):
         if unpack_force:
             self.unpack()
         _dest_file = os.path.normpath(self.get_file_path(file_name))
-        _dest_file = _dest_file if os.path.isfile(_dest_file) else None
+        _dest_file = _dest_file if os.path.isfile(_dest_file) and not re.match(".*txt.lock", _dest_file) else None
```

Comment:
```suggestion
        _dest_file = _dest_file if os.path.isfile(_dest_file) and not  _dest_file.endswith(".*txt.lock") else None
```
No matter to use regex here